src/purchase/serializers.py

```diff
@@ -394,3 +401,14 @@ def get_stats(self, purchase):
             "end_date": end_date,
         }
         return stats
+
+
+class RscExchangeRateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = RscExchangeRate
+        fields = "__all__"
+
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+        data["created_date"] = instance.created_date.isoformat()
+        return data
```

Comment:
Is this necessary?

Reply:
1. Without overriding the `to_representation` method ```"created_date": "2023-02-24T19:43:27.098203Z"```
2. Overriding the method ```"created_date": "2023-02-24T19:43:27.098203+00:00"```

Reply:
If that specific format is necessary, can we do this instead?

```
class RscExchangeRateSerializer(serializers.ModelSerializer):
    created_date = serializers.DateTimeField(format="%Y-%m-%dT%H:%M:%S.%f%z")

    class Meta:
        model = RscExchangeRate
        fields = "__all__"

```

This will return ```"created_date": "2023-02-24T19:43:27.098203+0000"```

Reply:
I'm OK removing. will do that then merge